mods/ra/rules/vehicles.yaml

```diff
@@ -733,6 +736,9 @@ QTNK:
 		RevealGeneratedShroud: False
 	RevealsShroud@GAPGEN:
 		Range: 4c0
+	RevealOnDeath:
```

Comment:
Not sure if that's what you said with the above, but it would be a nice detail if the Nuke Truck provided death vision only when _deliberately_ triggered/deployed.